homework/index.js

```diff
@@ -30,18 +30,132 @@
     return elem;
   }
 
+  function presentRepositories(repositoriesContainer, repository) {
+    const table = createAndAppend('table', repositoriesContainer);
+    const tBody = createAndAppend('tbody', table);
+    const headersOfDetails = [
+      'Repository:',
+      'Description:',
+      'Forks:',
+      'Open Issues:',
+      'Created:',
+      'Updated:',
+    ];
+    headersOfDetails.forEach(header => {
+      const tRow = createAndAppend('tr', tBody);
+      for (let j = 0; j < 2; j++) {
+        createAndAppend('td', tRow);
+        tRow.childNodes[0].innerText = header;
+      }
+    });
+    const tData = table.querySelector('tr').lastChild;
+    const a = createAndAppend('a', tData, {
+      href: repository.html_url,
+      target: '_blank',
+    });
+    a.textContent = repository.name;
+
+    const upDate = `${new Date(repository.updated_at).toLocaleDateString()}, ${new Date(
```

Comment:
Hi Ammar - this is just FYI but you can move this out to a function as well. Something like:
```
const getDate = date => `${new Date(date).toLocaleDateString()}, ${new Date(date).toLocaleTimeString()}`
```
and then use it:
```
const upDate = getDate(repository.updated_at);
const creation = getDate(repository.created_at)l
```